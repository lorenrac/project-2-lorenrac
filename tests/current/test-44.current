ERROR (line 2): Expected ')' after IF
