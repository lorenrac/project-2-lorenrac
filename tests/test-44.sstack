// ERROR: Don't allow non-associative chaining.
IF ("A" < "B" < "C") PRINT "Chaining too far..."
