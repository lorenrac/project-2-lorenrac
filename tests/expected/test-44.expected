ERROR (line 2): Cannot chain non-associative operators.
